source/authentication/email-password.txt

```diff
@@ -281,29 +277,29 @@ Send a Password Reset Email
 
 If you configure the provider to :guilabel:`Send a Password Reset Email`,
 {+service-short+} sends a Password Reset URL to a user's email address when a
-user requests a password reset. The user must visit this URL within 30 minutes
+{+service-short+} sends a Password Reset URL to the user's email address on a password reset request. The user must visit this URL within 30 minutes
 to confirm their intent to change their password.
 
-Configure the following settings to have {+service-short+} automatically send a
-password reset email:
+When you configure password reset emails, you can specify these settings:
+configure the following settings:
```

Comment:
Looks like there's some duplicated content here - let's fix this up by deleting ln 284.